src/Makefile.am

```diff
@@ -801,6 +801,15 @@ bitcoind_CXXFLAGS = $(bitcoin_bin_cxxflags)
 bitcoind_LDFLAGS = $(bitcoin_bin_ldflags)
 bitcoind_LDADD = $(LIBBITCOIN_NODE) $(bitcoin_bin_ldadd)
 
+#################################################################
+bin_PROGRAMS += ipv6-pinhole-test
```

Comment:
It might be useful for troubleshooting, though i'm not sure how happy people will be to add networking stuff to bitcoin-util. Currently it's pure-function stuff only 😄

Reply:
> You could make this a permanent debugging utility by adding it to bitcoin-util.

Concept NACK.

Reply:
Right. Let's not do this. Such a stand-alone tool, if it is useful, doesn't need to be part of bitcoin.